### ISSUE TITLE
No need for accounts JPY currency. 

### ISSUE THREAD

**Issue by elicoidal** [browseinfo/l10n_hk#1]
This is very specific and most of the time not needed (EUR and USD are more usual).

**Comment by Amit-Parik** [comment]
@bhavik-browseinfo : Please check this account , If Its not useful then please remove it and commit .

**Comment by bhavik-browseinfo** [comment]
Hello Eric!what do you mean ?accounts with JPY currency are not required ?or providing currency in those accounts is not required?There is a bit confusion here.

**Comment by elicoidal** [comment]
Hi Bhavik,
It means that the accounts for bank and receivable/payable are created 
for the JPY (Which is a strong currency but not the most used in HK). I 
think it is not necessary and only due to the fact that you got the 
original CoA from a Japanese implementation.
On the other hand I miss for receivable/payable and customer/supplier 
deposits the same accounts in EUR and USD (whereas they exist in JPY). 
This is more common in HK.
Another issue I just spotted: the accounts for currency should have the 
"Secondary currency" set up.

**Comment by lukebranch** [comment]
As a trading company in Hong Kong we deal a fair bit with CNY and TWD, as well as EUR and USD. 

I'd like to suggest that receivable/payable accounts with all of those currencies be implemented, as I think they are quite common in Hong Kong for trading companies (at least the ones in our industry). 

I guess the TWD and CNY accounts could be setup manually, as perhaps they are more specific use cases, but I definitely agree on the EUR and USD accounts.

For some reason the wizard defaults to EUR as well. I think as it is a HK COA and Localisation that it should probably default to HKD, but perhaps this is due to my setup, rather than due to the HK COA module itself.

**Comment by elicoidal** [comment]
CNY makes sense indeed. Not sure about TWD (similar to JPY).
I would stick to EUR/CNY/USD
Eric CAUDAL --
*Eric Caudal* /[Founder and CEO]/
Skype: elico.corp. Phone: + 86 186 2136 1670 (Cell), + 86 21 6211 
8017/27/37 (Office)
*Elico Shanghai* (Shenzhen/Singapore) <http://www.elico-corp.com> /*Odoo 
Gold Partner, best Odoo Partner 2014 for APAC*/

**Comment by lukebranch** [comment]
Agreed. We'll add TWD ourselves in our own fork, but CNY should definitely be included in the base localisation module.

**Comment by elicoidal** [comment]
Made some more tests today and here is my feedback:
Logo seems not correct (proportions)
Many accounts are misplaced / without correct parent
100001 and 100002 are created automatically depending on "1" should be under 1111

We still have many accounts for JPY: this is not necessary whereas many type of account do not have CNY, USD and EUR

111310 should be under parent 1113 Stocks
Missing/modifications:
111212 Accounts Receivable (Trade - EUR) 
111213 Accounts Receivable (Trade - CNY) 
111421 	Purchases Deposit Paid (USD)
111422 	Purchases Deposit Paid (EUR)
111423 	Purchases Deposit Paid (CNY)
 
121111 	Accounts Payable (Trade-USD) 
121112 	Accounts Payable (Trade-EUR) 
121113	Accounts Payable (Trade-CNY) 
121121 	Accounts Payable (N/T-USD)  
121122 	Accounts Payable (N/T-EUR)  
121123 	Accounts Payable (N/T-CNY) 
111411 	Prepayment (USD) =>Parent should be 1114
111412 	Prepayment (EUR) =>Parent should be 1114
111413 	Prepayment (CNY) =>Parent should be 1114

121310 => parent should be 1213
121410 => parent should be 1214
 
Besides, I install the hk as first module but I get the following which is wrong as the CoA for automated test is installed:
• test CoA is installed
• Test journal are installed
• Fiscal year naming is about test.

**Comment by bhavik-browseinfo** [comment]
Hello Eric!we have made changes in chart according to your suggestions.You can test with latest pool.
We have included currency in secondary currency of account wherever it is necessary. 
For installation without Test COA please do not include demo data while creating test database.
Again thanks for your valuable suggestion.
